src/main/java/net/minecraftforge/fml/common/FMLCommonHandler.java

```diff
@@ -202,7 +202,7 @@ public Side getSide()
     public Side getEffectiveSide()
     {
         Thread thr = Thread.currentThread();
-        if (thr.getName().equals("Server thread") || thr.getName().startsWith("Netty Server IO"))
+        if (thr.getName().equals("Server thread") || thr.getName().startsWith("Netty Epoll Server IO") || thr.getName().startsWith("Netty Server IO"))
```

Comment:
Such as?
Checking only for exact names is the reason this issue exists.
If accuracy is important, this method is the wrong way to approach the problem anyway.